.gitignore

```diff
@@ -1,6 +1,5 @@
 # compiled output
 /node_modules
-/lib
```

Comment:
This would need to be reverted. Everything under lib s should be ignored. The build is generated with CI